android/gradle.properties

```diff
@@ -12,10 +12,8 @@ org.gradle.jvmargs=-Xmx2048m -XX:MaxPermSize=512m -XX:+HeapDumpOnOutOfMemoryErro
 # org.gradle.parallel=true
 #Tue Dec 19 15:08:27 EST 2023
 KlaviyoReactNativeSdk_klaviyoAndroidSdkVersion=3.0.3
-KlaviyoReactNativeSdk_compileSdkVersion=31
-KlaviyoReactNativeSdk_kotlinVersion=1.8.0
+KlaviyoReactNativeSdk_kotlinVersion=1.7.0
```

Comment:
do we have to go down a kotlin version?

Reply:
Hm I didn't think much of this, but looking back, it seems we bumped that up from 1.7.0 in the past to 1.8.0 for RN versions <= 0.72.0 
Going to double check against #71 ...

Reply:
In that PR I wrote this (1.9.22 is the kotlin version of the android native SDK)

> 1.8 is compatible with 1.9.22 so we can stop here for now

Interesting because 1.7.0 is not causing any problems interfacing with our android SDK in my testing right now. I don't see any compelling reason to drop back to 1.7.0 though, so I will revert this line.

Reply:
going back to 1.8.0